core/Tracker/Visit/SpamFilter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Piwik\Tracker\Visit;
+
+use Piwik\Common;
+use Piwik\Tracker\Request;
+
+/**
+ * Filters out tracking requests issued by spammers.
+ */
+class SpamFilter
```

Comment:
could we rename class to `ReferrerSpam` since it's only a subset of possible Spam filtering